store.go

```diff
@@ -1175,7 +1180,26 @@ func (s *store) CreateContainer(id string, names []string, image, layer, metadat
 			},
 		}
 	}
-	clayer, err := rlstore.Create(layer, imageTopLayer, nil, "", nil, layerOptions, true)
+	if options.Flags == nil {
+		options.Flags = make(map[string]interface{})
+	}
+	if (options.Flags["ProcessLabel"].(string) != "" &&
+		options.Flags["MountLabel"].(string) == "") ||
+		(options.Flags["ProcessLabel"].(string) == "" &&
+			options.Flags["MountLabel"].(string) != "") {
```

Comment:
These type assertions need to be checked, in case any of these are unset.  When they're not set, we'll panic otherwise: https://play.golang.org/p/s_9RhYGbc8M.